src/tests/ftest/container/boundary.py

```diff
@@ -22,120 +22,130 @@ class BoundaryTest(TestWithServers):
     :avocado: recursive
     """
 
-    def create_container_and_test(self, pool=None, pool_num=1, cont_num=1, with_io=False):
+    def setUp(self):
+        """Set Up BoundaryTest"""
+        super().setUp()
+        self.pool = []
+
+    def create_container_and_test(self, pool=None, cont_num=1, with_io=False):
         """To create single container on pool.
+
         Args:
             pool (str): pool handle to create container.
-            pool_num (int): pool number.
             container_num (int): container number to create.
             with_io (bool): enable container test with execute_io.
-        """
 
+        """
         try:
             container = self.get_container(pool)
         except (DaosTestError, TestFail) as err:
-            msg = "#(3.{}.{}) container create failed. err={}".format(pool_num, cont_num, err)
+            msg = "#(3.{}.{}) container create failed. err={}".format(pool.label, cont_num, err)
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string